### ISSUE TITLE
Tensorflow-macos and tensorflow-quantum

### ISSUE THREAD

**Issue by benzabonanza** [tensorflow/quantum#682]
Hi there,

I get the following error when I try to install tensorflow-quantum in my conda environment:

ERROR: Could not find a version that satisfies the requirement tensorflow-quantum (from versions: none)
ERROR: No matching distribution found for tensorflow-quantum

I finally managed to put **tensorflow-macos==2.7.0** on my M1 Mac such that it doesn't clashes with the hardware when I import it.
But I can't find any way to install tensorflow-quantum, at least not with **pip3 install -U tensorflow-quantum** or some variation of it. I was wondering if it is because I don't have tensorflow==2.7.0 but tensorflow-macos==2.7.0. 

Package list in the environment attached in the env.txt file below. Working with miniforge channel as it was suggested for M1 Tensorflow.

I hope some of you might have an idea how to end my misery here.

[env.txt](https://github.com/tensorflow/quantum/files/8462796/env.txt)
(I can also just write down package versions etc but just didn't put the whole thing here)

**Comment by lockwo** [comment]
I don't have a M1 mac so I can't verify this but https://github.com/tensorflow/quantum/issues/655 seems to have good steps for it and https://github.com/tensorflow/quantum/issues/492 as to why this may be the case.

**Comment by tembence** [comment]
Thanks Owen, I missed that post. I will try that way... quite annoying, it took me some time to make TF run natively on M1. With this method I have to forget about all that and use Rosetta.

Then the question is still open for the future: 
"How to install TFQ on M1, i.e. with tensorflow-macos?"

**Comment by lockwo** [comment]
Yes, I don't think the software has fully caught up to the M1 (I have heard a lot of horror stories of installing various programs).
